### DISCUSSION TITLE
三次编译后书签仅有一级标题

### DISCUSSION THREAD

**Discussion by XDUXTU** [note286/xduts#49] [Q&A] [answered]
请问当使用跨页表格时要生成正确的格式必须进行三次Xelatex编译，但是三次编译之后导航栏的目录(非正文里的目录)只剩下了章节标题，该章节下的小节目录消失了，从导航栏中访问章节里的小节就很麻烦，有什么办法能够解决吗？
<img width="94" alt="issue" src="https://user-images.githubusercontent.com/125463275/219938611-6dafa819-cba4-4f9d-adc5-2faee528972a.png">

**Comment by note286** [comment, marked as answer]
请更新至最新版。

**Reply by XDUXTU** [reply]
谢谢大佬，更新后解决了
